thirdeye/thirdeye-frontend/app/pods/home/controller.js

```diff
@@ -32,5 +43,17 @@ export default Controller.extend({
 
       return statsArray;
     }
-  )
+  ),
+
+  actions: {
+
+    /**
+     * Sets the selected application property based on user selection
+     * @param {Object} selectedApplication - object that represents selected application
+     * @return {undefined}
+     */
+    selectApplication(selectedApplication) {
+      setProperties(this, { selectedApplication });
```

Comment:
Should we use only `set` here if it's only for one prop?

Reply:
I figured it would be shorter than: 
`set (this, 'selectedApplication', this.selectedApplication)`

What do you think?